install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -73,8 +74,10 @@ spec:
               endpoint:
                 properties:
                   external:
+                    description: The external endpoint address that applications can use to connect to the Broker
```

Comment:
...external to the cluster within which the Broker is running.

Reply:
Done.